sdk/python/tests/compiler/testdata/resourceop_basic.log

```diff
@@ -0,0 +1,5 @@
+[test-step : main] time="2020-06-09T02:49:02Z" level=error msg="Initialize script failed: exit status 2:"
+[test-step : main] time="2020-06-09T02:49:02Z" level=info msg="kubectl create -f /tmp/manifest.yaml -o json"
+[test-step : main] time="2020-06-09T02:49:03Z" level=error msg="Run kubectl command failed with: exit status 1 and Error from server (Forbidden): error when creating \"/tmp/manifest.yaml\": jobs.batch is forbidden: User \"system:serviceaccount:kubeflow:default\" cannot create resource \"jobs\" in API group \"batch\" in the namespace \"kubeflow\""
+[test-step : main] time="2020-06-09T02:49:03Z" level=error msg="Execute resource failed: exit status 1:"
```

Comment:
By default, KFP doesn't allow any resource to be created by pipeline. The API has an option for users to provide their own service account that has permissions to do these kind of action. To run it with Tekton alone, please use these instructions https://github.com/kubeflow/kfp-tekton/blob/master/sdk/sa-and-rbac.md

Reply:
All the resourceOp and volumeOp probably needs to cleanup because they will be creating new resources every time you run the test.

Reply:
I added a TODO for the post-test cleanup and for a pre-test setup. I will create a separate issue to track those